ckan fails to run can't parse ksp version

<!--

Thanks for wanting to bring an issue to our attention!

Before you open a ticket, please do a quick search of the existing issues to see if it's already been brought up.

https://github.com/KSP-CKAN/CKAN/issues?q=is%3Aissue

-->

Background
----------

**CKAN Version:** 
v1.26.0+542147ea543

**KSP Version:** 
1.6.1.2401 (LinuxPlayer)

**Operating System:** 
Void Linux

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?** 
nope

Problem
-------

**What steps did you take in CKAN?**
ran it

**What did you expect to happen?**
it should run

**What happened instead?**
it didn't

**Screenshots:**
<!-- You can copy an image of a window in Windows with Alt-PrtScr, then paste it in GitHub with ctrl-V -->


**CKAN error codes (if applicable):** 

Unhandled Exception:
System.ArgumentNullException: Value cannot be null.
Parameter name: input
  at CKAN.Versioning.KspVersion.Parse (System.String input) [0x00009] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.KSP+<>c.<LoadCompatibleVersions>b__25_0 (System.String v) [0x00000] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at System.Linq.Enumerable+SelectListIterator`2[TSource,TResult].ToList () [0x0002a] in <eb5d259bb9404e2799b2425f777626f7>:0 
  at System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x0001f] in <eb5d259bb9404e2799b2425f777626f7>:0 
  at CKAN.KSP.LoadCompatibleVersions () [0x0004c] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.KSP..ctor (System.String gameDir, System.String name, CKAN.IUser user) [0x00046] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.KSPManager.LoadInstancesFromRegistry () [0x0006b] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.KSPManager..ctor (CKAN.IUser user, CKAN.IWin32Registry win32_registry) [0x0002c] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.CmdLine.MainClass.Execute (CKAN.KSPManager manager, CKAN.CmdLine.CommonOptions opts, System.String[] args) [0x00178] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x000a1] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.ArgumentNullException: Value cannot be null.
Parameter name: input
  at CKAN.Versioning.KspVersion.Parse (System.String input) [0x00009] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.KSP+<>c.<LoadCompatibleVersions>b__25_0 (System.String v) [0x00000] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at System.Linq.Enumerable+SelectListIterator`2[TSource,TResult].ToList () [0x0002a] in <eb5d259bb9404e2799b2425f777626f7>:0 
  at System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x0001f] in <eb5d259bb9404e2799b2425f777626f7>:0 
  at CKAN.KSP.LoadCompatibleVersions () [0x0004c] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.KSP..ctor (System.String gameDir, System.String name, CKAN.IUser user) [0x00046] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.KSPManager.LoadInstancesFromRegistry () [0x0006b] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.KSPManager..ctor (CKAN.IUser user, CKAN.IWin32Registry win32_registry) [0x0002c] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.CmdLine.MainClass.Execute (CKAN.KSPManager manager, CKAN.CmdLine.CommonOptions opts, System.String[] args) [0x00178] in <23e46ed2c17345769e5f22ae7dbd9d30>:0 
  at CKAN.CmdLine.MainClass.Main (System.String[] args) [0x000a1] in <23e46ed2c17345769e5f22ae7dbd9d30>:0

--- a/GUI/CompatibleKspVersionsDialog.cs
+++ b/GUI/CompatibleKspVersionsDialog.cs
@@ -86,6 +86,11 @@ namespace CKAN
             {
                 return;
             }
+            if (AddVersionToListTextBox.Text.ToLower() == "any") 
+            {
+                MessageBox.Show("Version has invalid format", "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                return;
+            }
             try
             {
                 var version = KspVersion.Parse(AddVersionToListTextBox.Text);

